src/messages/formatters.ts

```diff
@@ -0,0 +1,154 @@
+export function codeBlock(language: string, content?: string): string {
+  return typeof content === 'undefined' ? `\`\`\`\n${language}\`\`\`` : `\`\`\`${language}\n${content}\`\`\``;
+}
+
+/**
+ * Wraps the content inside an inline code.
+ * @param content The content to wrap.
+ * @returns The formatted content.
+ */
+export function inlineCode<C extends string>(content: C): `\`${C}\`` {
+  return `\`${content}\``;
+}
+
+/**
+ * Formats the content into italic text.
+ * @param content The content to wrap.
+ * @returns The formatted content.
+ */
+export function italic<C extends string>(content: C): `*${C}*` {
+  return `*${content}*`;
+}
+
+/**
+ * Formats the content into bold text.
+ * @param content The content to wrap.
+ * @returns The formatted content.
+ */
+export function bold<C extends string>(content: C): `**${C}**` {
+  return `**${content}**`;
+}
+
+/**
+ * Formats the content into underscored text.
+ * @param content The content to wrap.
+ * @returns The formatted content.
+ */
+export function underscore<C extends string>(content: C): `__${C}__` {
+  return `__${content}__`;
+}
+
+/**
+ * Formats the content into strikethrough text.
+ * @param content The content to wrap.
+ * @returns The formatted content.
+ */
+export function strikethrough<C extends string>(content: C): `~~${C}~~` {
+  return `~~${content}~~`;
+}
+
+/**
+ * Formats the content into a quote. This needs to be at the start of the line for Discord to format it.
+ * @param content The content to wrap.
+ * @returns The formatted content.
+ */
+export function quote<C extends string>(content: C): `> ${C}` {
+  return `> ${content}`;
+}
+
+/**
+ * Formats the content into a block quote. This needs to be at the start of the line for Discord to format it.
+ * @param content The content to wrap.
+ * @returns The formatted content.
+ */
+export function blockQuote<C extends string>(content: C): `>>> ${C}` {
+  return `>>> ${content}`;
+}
+
+/**
+ * Formats a date into a short date-time string.
+ * @param date The date to format, defaults to the current time.
+ */
+export function time(date?: Date): `<t:${bigint}>`;
+/**
+ * Formats a date given a format style.
+ * @param date The date to format.
+ * @param style The style to use.
+ */
+export function time<S extends TimestampStylesString>(date: Date, style: S): `<t:${bigint}:${S}>`;
+/**
+ * Formats the given timestamp into a short date-time string.
+ * @param seconds The time to format, represents an UNIX timestamp in seconds.
+ */
+export function time<C extends number>(seconds: C): `<t:${C}>`;
+/**
+ * Formats the given timestamp into a short date-time string.
+ * @param seconds The time to format, represents an UNIX timestamp in seconds.
+ * @param style The style to use.
+ */
+export function time<C extends number, S extends TimestampStylesString>(seconds: C, style: S): `<t:${C}:${S}>`;
+export function time(timeOrSeconds?: number | Date, style?: TimestampStylesString): string {
```

Comment:
```suggestion
export function time<C extends number, S extends TimestampStylesString>(seconds: C, style: S): `<t:${C}:${S}>`;

export function time(timeOrSeconds?: number | Date, style?: TimestampStylesString): string {
```

Unspaced TSDoc wasn't enough so you decided to unspace functions too?

Reply:
Missing TSDoc

Reply:
The implementation function never has docs, the overloads do.